frontend/src/features/public-form/components/FormFields/FormFields.tsx

```diff
@@ -74,6 +80,27 @@
 
   const defaultFormValues = useMemo(() => {
     return augmentedFormFields.reduce<FormFieldValues>((acc, field) => {
+      // If this is part of a MRF flow, use that.
+      const previousResponse = previousResponses?.[field._id]
+      if (previousResponse) {
+        switch (field.fieldType) {
+          case BasicField.CountryRegion: {
+            const selected = Object.values(CountryRegion).find(
+              (option) => option.toUpperCase() === previousResponse.answer,
+            )
+            if (selected) {
+              acc[field._id] = selected
+            }
+            break
+          }
+          case BasicField.Attachment:
+          //TODO(MRF): Handling of attachments by respondent 2+
+          default:
+            acc[field._id] = previousResponse.answer as FormFieldValue
+        }
+        return acc
```

Comment:
Q: This means that MPF responses will now take precedence over prefills. Prefills will still work for first respondents but subsequent respondents will not see any prefills

Reply:
correct! i presume this would be intended - we should investigate how prefills would work with MRF responses though. Will open a ticket for this, for now users should not use MRF with prefills for pilot